Rethinking_2/End_of_chapter_problems/Chapter 3 - SAMPLING THE IMAGINARY.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>add a semicolon at the end of the last line like <code>plt.legend(loc=(1.1,0.5));</code></p><p> </p><p>This supress the matplotlib output (the <code>&lt;matplotlib.legend.Legend at 0x7fea4033c3d0&gt;</code>  )in this example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>